ember-bootstrap/addon/utils/is-fastboot.ts

```diff
@@ -0,0 +1,11 @@
+import { getOwner } from '@ember/application';
+import type FastBootService from 'ember-cli-fastboot/services/fastboot';
+
+export default function isFastBoot(context: object) {
+  const owner = getOwner(context);
+  if (!owner) {
+    return false;
+  }
+  const fastbootService = owner.lookup('service:fastboot') as FastBootService;
```

Comment:
I think the type is not fully correct here. `Owner.lookup` returns `undefined` or `null` (cannot recall anymore) if no factory has been registered with that name. Sadly the return type in that case doesn't seem to be documented well by Ember.

Reply:
The null check after that makes more sense like that. I'll update it.